pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -123,30 +123,12 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 				}: 1,
 			},
 		},
-		{
-			name: "parse_frames tail call using 2 programs",
-			// The purpose of this test is to validate that when we surpass the limit of HTTP2_MAX_FRAMES_ITERATIONS,
-			// the filtering of subsequent frames will continue using tail calls.
-			messageBuilder: func() []byte {
-				const settingsFramesCount = 130
-				return createMessageWithCustomSettingsFrames(t, testHeaders(), settingsFramesCount)
-			},
-			expectedEndpoints: map[http.Key]int{
-				{
-					Path:   http.Path{Content: http.Interner.GetString("/aaa")},
-					Method: http.MethodPost,
-				}: 1,
-			},
-			// Currently we don't have a way to test it as TLS version does not have the ability to run 2 tail calls
-			// for filtering frames (will be fixed in USMON-684)
-			skip: s.isTLS,
-		},
 		{
 			name: "validate frames_filter tail calls limit",
 			// The purpose of this test is to validate that when we surpass the limit of HTTP2_MAX_TAIL_CALLS_FOR_FRAMES_FILTER,
 			// for 2 filter_frames we do not use more than two tail calls.
```

Comment:
please correct the ut's documentation as well as we now not using the tail calls